internal/commands/build/types.go

```diff
@@ -70,3 +76,30 @@ func transformBuildConfig(data interface{}) (interface{}, error) {
 		return data, errors.Errorf("invalid type %T for service build", value)
 	}
 }
+
+func buildArgsToMap(array []string) map[string]string {
+	result := make(map[string]string)
+	for _, value := range array {
+		parts := strings.SplitN(value, "=", 2)
+		key := parts[0]
+		switch {
```

Comment:
Why a switch here and not a
```go
if len(parts) == 1 {
         result[key] = ""
} else {
        result[key] = parts[1]
}
```
that does the same thing but maybe more easy to read

Reply:
changed 😉

Reply:
I'm fine with switch 😅 
`else` is bad, mkay?

Reply:
If `else` is bad you can write:

```go
result[key] = ""
if len(parts) != 1 {
        result[key] = parts[1]
}
```